edx_proctoring/api.py

```diff
@@ -1586,6 +1629,14 @@ def update_attempt_status(attempt_id, to_status,
     # we use the 'status' field as the name of the event 'verb'
     emit_event(exam, attempt['status'], attempt=attempt)
 
+    if (to_status == ProctoredExamStudentAttemptStatus.verified and exam['is_proctored']):
+        # If the user has a verified name entry linked to this exam attempt, approve it
+        user = USER_MODEL.objects.get(id=user_id)
+        try:
+            update_is_verified_status(user, True, proctored_exam_attempt_id=attempt_id)
+        except VerifiedNameDoesNotExist:
+            pass
+
```

Comment:
this is not really something you can solve in this PR but I was reading back to see if I could understand this and update_attempt_status could use some chopping up, it's huge